client/src/components/Profile/EditProfile.tsx

```diff
@@ -0,0 +1,238 @@
+                  value={values.firstName}
+                  onChange={handleChange}
+                />
+              </Grid>
+
+              <Grid item xs={4}>
+                <h3 className={classes.labelStyle}>LastName </h3>
+              </Grid>
+              <Grid item xs={8}>
+                <TextField
+                  name="lastName"
+                  className={classes.textInputStyle}
+                  onChange={handleChange}
+                  label="LastName"
+                  variant="outlined"
+                  value={values.lastName}
+                />
+              </Grid>
+
+              <Grid item xs={4}>
+                <h3 className={classes.labelStyle}>Gender </h3>
+              </Grid>
+              <Grid item xs={8}>
+                <Autocomplete
+                  id="gender"
+                  value={values.gender}
+                  className={classes.textInputStyle}
+                  onChange={(c, v) => setFieldValue('gender', v)}
+                  options={gender}
+                  getOptionLabel={(option) => option}
+                  renderInput={(params) => <TextField {...params} label="Gender" variant="outlined" />}
+                />
+              </Grid>
+              <Grid item xs={4}>
+                <h3 className={classes.labelStyle}>Birthday</h3>
+              </Grid>
+              <Grid item xs={8}>
+                <TextField
+                  name="birthDate"
+                  type="date"
+                  className={classes.textInputStyle}
+                  onChange={handleChange}
+                  variant="outlined"
+                  value={moment(values.birthDate).format('YYYY-MM-DD')}
+                />
+                {/* <div style={{ display: 'flex', width: '75%' }}>
+                  <Autocomplete
+                    id="date"
+                    className={classes.textInputStyle}
+                    onChange={(c, v) => setFieldValue('birthday', v)}
+                    options={months}
+                    getOptionLabel={(option) => option}
+                    renderInput={(params) => <TextField {...params} variant="outlined" />}
+                  />
+                  <Autocomplete
+                    id="date"
+                    className={classes.textInputStyle}
+                    onChange={(c, v) => setFieldValue('birthday', v)}
+                    options={months}
+                    getOptionLabel={(option) => option}
+                    renderInput={(params) => <TextField {...params} variant="outlined" />}
+                  />
+                  <Autocomplete
+                    id="date"
+                    className={classes.textInputStyle}
+                    onChange={(c, v) => setFieldValue('birthday', v)}
+                    options={months}
+                    getOptionLabel={(option) => option}
+                    renderInput={(params) => <TextField {...params} variant="outlined" />}
+                  />
```

Comment:
Please remove this if it's not being used, we generally don't want commented code pushed to production.